tests/test-sources/plugins/by-name/lsp_signature/default.nix

```diff
@@ -0,0 +1,33 @@
+{
+  empty = {
+    plugins.lsp_signature.enable = true;
+  };
+
+  example = {
+    plugins.lsp_signature = {
+      enable = true;
+
+      settings = {
+        debug = true;
+        log_path = /home/wadsaek/.config/lsp_signature.log;
```

Comment:
When you use an unquoted path like this, nix actually copies it into the nix-store before evaluating the flake. It also complains if the path is outside of the flake's tree:

```
… while evaluating definitions from `/nix/store/XXX-source/plugins/by-name/lsp_signature':
… while evaluating an attribute `log_path`
error: access to absolute path '/home' is forbidden in pure evaluation mode
```

Ideally, this should be set to something that can actually be used when running the test-suite.

OTTOMH, I guess it should be possible to write to `/tmp/*`? But you'd have to pass that as a string to avoid the eval error.

Reply:
if it's passed as a string, then any writeable path would work i think, but yeah i totally forgot that nix copies all the paths to the store, thanks for reminding